provision/acc_provision/acc_provision.py

```diff
@@ -494,6 +479,43 @@ def config_adjust(args, config, prov_apic, no_random):
         }
     }
 
+    for i, child in enumerate(adj_config):
+        if "kube_config" in child.keys():
+            kube_config_object=adj_config["kube_config"]
+            for pod_subnet in pod_subnets:
+                kube_config_object.append(
+                    "pod_ip_pool":
+                        [
+                          {
+                            "start": cidr_split(pod_subnet)[0],
+                            "end": cidr_split(pod_subnet)[1],
+                          }
+                        ]
+
+                    "pod_network":
+                        [
+                          {
+                            "subnet": "%s/%s" % cidr_split(pod_subnet)[3:5],
+                            "gateway": cidr_split(pod_subnet)[2],
+                            "routes": [
+                                {
+                                    "dst": config_set_dst(pod_subnet),
+                                    "gw": cidr_split(pod_subnet)[2],
+                                }
+                             ]
+                          }
+                        ]
+                )
```

Comment:
This will result in multiple pod_ip_pool and pod_network key:value items in the kube_config_object. Instead we need to add each subnet as a new entry in each of the lists corresponding to the values of the keys "pod_ip_pool" and "pod_network", e.g.:

     "pod_ip_pool": [ {}, {}, ..]

Similarly, in the net_config_object below.

Also, there is a "," missing between the items "pod_ip_pool" and "pod_network".